tabular/src/autogluon/tabular/models/tab_transformer/hyperparameters/searchspaces.py

```diff
@@ -0,0 +1,15 @@
+from autogluon.core import Real, Categorical, Int
+
+# TODO: May have to split search space's by problem type. Not necessary right now.
+def get_default_searchspace():
+    params = {
+        'lr': Real(5e-5, 5e-3, default=1e-3, log=True),
+        'weight_decay': Real(1e-6, 5e-2, default=1e-6, log=True),
+        'p_dropout': Categorical(0, 0.1, 0.2, 0.3, 0.4, 0.5),
+        'n_heads': Categorical(2, 4, 8),
+        'hidden_dim': Categorical(32, 64, 128, 256),
+        'n_layers': Categorical(1, 2, 3, 4, 5),
```

Comment:
sorry this wasn't clearly documented, but first value in the list of options is the default value for Categorical hyperparameters. So these should be re-ordered such that first value matches the value in `parameters.py`

Reply:
Updated the Categorical's to have first value be default.